salt_stack/salt/lambda-dev/init.sls

```diff
@@ -4,21 +4,14 @@
 
 {% set user = 'ec2-user' %}
 {% set venv_home = '/home/' + user + '/lambdaenv' %}
-{% set spdb_home = venv_home + '/usr/lib/python3.6/site-packages/spdb' %}
-{% set bossutils_home = venv_home + '/usr/lib/python3.6/site-packages/bossutils' %}
-{% set lambda_home = venv_home + '/usr/lib/python3.6/site-packages/lambda' %}
-{% set lambdautils_home = venv_home + '/usr/lib/python3.6/site-packages/lambdautils' %}
+{% set spdb_home = venv_home + '/usr/lib/python3.7/site-packages/spdb' %}
+{% set bossutils_home = venv_home + '/usr/lib/python3.7/site-packages/bossutils' %}
+{% set lambda_home = venv_home + '/usr/lib/python3.7/site-packages/lambda' %}
+{% set lambdautils_home = venv_home + '/usr/lib/python3.7/site-packages/lambdautils' %}
 
 include:
     - node
-
-python36:
-    pkg.installed:
-        - pkgs:
-            - python36
-            - python36-pip
-            - python36-virtualenv
-
+    - python.python37
```

Comment:
What OS are we running on the lambda build server now? Is it possible to get 3.7 with the OS' package manager?

Reply:
It's amazon linux.  I looked they didn't have a 3.7